test/integration/test-extensions-loading.js

```diff
@@ -21,8 +21,6 @@ function checkElementUpgrade(element) {
   expect(element).to.have.class('-amp-element');
   expect(element).to.have.class('-amp-layout-responsive');
   expect(element).to.have.class('-amp-layout-size-defined');
-  expect(element).to.not.have.class('-amp-notbuilt');
```

Comment:
Yes sorry. So @zhouyx here is testing only if the element was `upgraded` and so checking if it doesn't have `-amp-unresolved` should be enough. Our theory is that the event the test is waiting on is fired maybe too early before the iframe gets built and we weren't able to wait on `amp:load:start` to fire for the `iframe` since it might not be laid out because it probably is out of the viewport during the test.

So we opted for just dropping testing the build part in this test and just test upgrade which what the test is intended for.